spec/js/contact_us_form_element_spec.ts

```diff
@@ -93,4 +93,81 @@ describe('ContactUsFormElement', () => {
       });
     });
   });
+
+  describe('when there is no outage', () => {
```

Comment:
I think the behaviors here would be reasonably well-covered already by the "when maintenance window is not specified" test case above. The "when there is an unplanned outage" test below seems like what we'd want, to ensure that the form is hidden.